app/controllers/introduce_yourself_controller.rb

```diff
@@ -17,6 +17,10 @@ def update
 
   private
 
+  def ensure_application_present
+    _application_not_created = true
+  end
```

Comment:
It's not a problem a current snap application exists, that's fine. It's when it doesn't exist yet (which is only on the first visit to this page).